drivers/gps/nrf9160_gps/Kconfig

```diff
@@ -40,6 +40,18 @@ config NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 
 if NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 
+choice NRF9160_GPS_ANTENNA
+	prompt "GPS antenna selection"
+	default NRF9160_GPS_ANTENNA_INTERNAL
+
+config NRF9160_GPS_ANTENNA_INTERNAL
```

Comment:
Maybe use the word ONBOARD instead of INTERNAL as in the description.

Reply:
Agreed, changed to ONBOARD